utils/gitlog2changelog.py

```diff
@@ -15,7 +16,20 @@
     rev_range = "%s..HEAD" % base
 
 # Execute git log with the desired command line options.
-fin = os.popen("git log --summary --stat --no-merges --date=short %s" % rev_range, "r")
```

Comment:
What was the `r` argument before?

Reply:
"r" refers to the return value's read/write state. In this case, it's read only using os.popen

Reply:
Good!